### ISSUE TITLE
Issue in computing expressions

### ISSUE THREAD

**Issue by garg-ketki** [uber/NullAway#167]
```java
public enum TestEnum {
    VALUE1,
    VALUE2
  }

List<TestObject> testList = new ArrayList();

class TestObject {
    private TestEnum testEnum;
    public TestEnum getTestEnum() {
        return testEnum;
   }
}

if (testList.get(0).getTestEnum() != null) {
      return testList.get(0).getTestEnum().ordinal(); // not working
}

TestEnum testEnum = testList.get(0).getTestEnum(); // worked
if (testEnum != null) {
      return testEnum.ordinal();
}
```

Library version: 0.4.4

**Comment by msridhar** [comment]
@lazaroclapp #285 will fix this right?

**Comment by lazaroclapp** [comment]
Yes. This should be fixed now after #285 landed. We aren't cutting a release yet, I think, since that change + optional support by default should merit a `0.7.0` release. In the meantime, closing this issue. Feel free to re-open if not fixed on master or `0.7.0` when it is out.
